pythran/tests/test_random.py

```diff
@@ -31,6 +31,12 @@ def test_randint(self):
     def test_sample_(self):
         self.run_test("def sample_(n,k): from random import sample ; s = sum(sum(sample(range(n),k)) for x in range(n)) ; return abs(s/float(n*n)) < .05  ", 10**4, 4, sample_=[int, int])
 
+    def test_shuffle1(self):
+        self.run_test("def shuffle1(n): from random import shuffle ; r = range(n); shuffle(r); return len(r) == n and r != range(n)" , 10**4,  shuffle1=[int])
+
+    def test_shuffle2(self):
+        self.run_test("def shuffle2(n): from random import shuffle, random ; r = range(n); shuffle(r, lambda: random()); return len(r) == n and r != range(n);" , 10**4,  shuffle2=[int])
```

Comment:
It works too. I tried first lambda: 0, but both python and cpp implementations will differ (one seems to start from the beginning of the list and the other from the end), so I replaced 0 with `random()`, and forgot to remove the `lambda`.